test/test_with_xpress.jl

```diff
@@ -1027,3 +1027,34 @@ end
 
 end
 
+@testset "PVspecs" begin
+    ## Scenario 1: Palmdale, CA; array-type = 0 (Ground-mount)
+    post_name = "pv.json" 
+    post = JSON.parsefile("./scenarios/$post_name")
+    scen = Scenario(post)
+    
+    print("\nlatitude: ", post["Site"]["latitude"])
```

Comment:
please remove these print statements

Reply:
addressed in https://github.com/NREL/REopt.jl/pull/71/commits/ed4eb816434cf961579a2b827c29525a5764c61c